lib/streamlit/config_util.py

```diff
@@ -113,6 +114,7 @@ def append_setting(text):
                 # # Add a line break after a paragraph only if it's not the last paragraph
                 if i != last_paragraph_idx:
                     out.append("")
+            import toml
```

Comment:
nit: would be nice to have a newline or two before the import

Reply:
I added one line (the maximum that the formatter allows)